types/propEq.d.ts

```diff
@@ -1,6 +1,29 @@
-export function propEq<T>(val: T): {
-  <K extends PropertyKey>(name: K): (obj: Record<K, T>) => boolean;
-  <K extends PropertyKey>(name: K, obj: Record<K, T>): boolean;
+import { Placeholder } from 'ramda';
+
+export function propEq(__: Placeholder): never;
```

Comment:
This makes sense to me, however I'm not sure how necessary it it

Reply:
It might be a bit extra, but it can save from misusing the placeholder